README.md

```diff
@@ -26,7 +26,42 @@
 The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
 also a live [Google Calendar][] that may be subscribed to.
 
-# LTS Plan
+## Mandate
+
+The Release and LTS working group's purpose is:
```

Comment:
The WG should be made up of two distinct teams: The Release Team and the LTS/Backporting Team. Everything here should be cast in terms of splitting the responsibilities of the WG between those two teams